policyengine_us/parameters/gov/states/ar/tax/income/rates/main.yaml

```diff
@@ -0,0 +1,47 @@
+description: Arkansas taxes personal income at these rates.
+metadata:
+  type: marginal_rate
+  threshold_unit: currency-USD
+  rate_unit: /1
+  label: Arkansas income tax rate
+  reference:
+  - title: Tax Legislation for 2022 Ark. Code § 26-51-201 (a)(1)(A)
+    href: https://casetext.com/statute/arkansas-code-of-1987/title-26-taxation/subtitle-5-state-taxes/chapter-51-income-taxes/subchapter-2-imposition-of-tax/section-26-51-201-effective-90-days-after-sine-die-adjournment-individuals-trusts-and-estates
+
+  - title: 2022 Indexed Tax Brackets
+    href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2022.pdf
```

Comment:
need to add pdf paged in this format
https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/TaxBrackets_2022.pdf#page=x

Reply:
all pdf sources and files